src/ch/imedias/rsccfx/view/RsccRequestView.java

```diff
@@ -19,11 +19,12 @@
  * Defines all elements shown in the request section.
  */
 public class RsccRequestView extends BorderPane {
-  private final Rscc model;
-  private final Strings strings = new Strings();
+  private static final double STANDARD_SIZE = 50d;
+  private static final double GENERATEKEYFIELD_HEIGHT = 60d;
```

Comment:
I think maybe it would make sense here to call it according to the variable name of the text field, so it would be: "GENERATEDKEYFLD_HEIGHT"?